File: src/main/java/com/hsmchurch/app/noticeBoard/NoticeRepository.java
package com.hsmchurch.app.noticeBoard;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QuerydslPredicateExecutor;

public interface NoticeRepository extends JpaRepository<Notice, Long>, QuerydslPredicateExecutor<Notice> {

}

File: src/main/java/com/hsmchurch/app/noticeboard/domain/NoticeRepository.kt
package com.hsmchurch.app.noticeboard.domain

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QuerydslPredicateExecutor

interface NoticeRepository : JpaRepository<Notice, Long>, QuerydslPredicateExecutor<Notice>, NoticeRepositoryCustom
